addons/website/static/tests/tours/page_manager.js

```diff
@@ -178,6 +228,7 @@ registerWebsitePreviewTour('website_page_manager', {
     },
     ...checkKanbanGroupBy,
     ...checkWebsiteFilter,
+        ...checkActiveWebsiteFilter,
```

Comment:
```suggestion
    ...checkActiveWebsiteFilter,
```

Reply:
The linter suggests this spacing. Should we use it?